build/gulp/tasks/docs.ts

```diff
@@ -55,6 +61,7 @@ task(
 // ----------------------------------------
 
 const componentsSrc = [`${config.paths.src()}/components/*/[A-Z]*.tsx`]
+const behaviorSrc = [`${config.paths.src()}/lib/accessibility/Behaviors/*/[A-Z]*.ts`]
```

Comment:
What about `DefaultBehavior.ts`?

Reply:
this is not real behavior which would need documentation, so it doesn't need be included